Dockerfile

```diff
@@ -32,20 +32,19 @@ ARG DISTROLESS_IMAGE=gcr.io/distroless/static@sha256:c6d5981545ce1406d33e61434c6
 # libc, muscl, etc.
 FROM ${GOLANG_IMAGE} as builder
 
-# This build arg is the version to embed in the CPI binary
-ARG VERSION=unknown
-
-# This build arg controls the GOPROXY setting
-ARG GOPROXY
+ARG VERSION
+ARG GOPROXY=https://goproxy.io,direct
+ARG GOOS=linux
 
 WORKDIR /build
 COPY go.mod go.sum ./
 COPY cmd/ cmd/
 COPY pkg/ pkg/
 COPY Makefile ./Makefile
-ENV CGO_ENABLED=0
-ENV GOPROXY ${GOPROXY:-https://proxy.golang.org}
-RUN make aws-cloud-controller-manager
+RUN GO111MODULE=on CGO_ENABLED=0 GOOS=${GOOS} GOPROXY=${GOPROXY} go build \
```

Comment:
Any specific reason why we're not simply running make anymore?

Reply:
I didn't like the circular dependency of make -> dockerfile -> make.  It makes the env var/docker args logic hard to follow IMO.

Reply:
I see. I was more concerned about changing the build command on Makefile and then forgetting to update it here. I'm not super familiar with the best practices here, so I'll leave it to your judgement.

Reply:
Yeah I considered the downside but I don't think the command will change much at all